RecoTracker/LSTCore/src/alpaka/Quintuplet.h

```diff
@@ -1798,20 +1798,20 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
       }
       alpaka::syncBlockThreads(acc);
 
-      // Occupancy matrix for 0.8 GeV pT Cut
+      // Occupancy matrix for 0.8 GeV pT Cut, 99.99%
       constexpr int p08_occupancy_matrix[4][4] = {
-          {336, 414, 231, 146},  // category 0
+          {161, 129, 135, 132},  // category 0
```

Comment:
this is a pretty large drop: does this rely on the pending T3 DNN?

Reply:
No, it shouldn't. I was surprised by this as well, I will check to make sure I wasn't on the wrong branch. Maybe The first two categories use a higher percentile?

Reply:
Also, if you look at the old .06 pT matrix below, it has a value of 325 at 99.99% which implies what I said above about the first two maybe originating from a higher percentile. 161 and 509 make a lot more sense for 0.8 and 0.6 than 336 and 325.

Reply:
I also don't think the T3 DNN will affect the T5 occupancies much. The T5 fake rate is already pretty low compared to the T3 fake rate, so a cleaner set of T3 candidates shouldn't lower the T5 occupancies much.

Reply:
<img src="https://github.com/user-attachments/assets/54250b23-bb27-43a6-bb6b-f60672031a6c" width="300" height="auto" alt="Screenshot 4">

Percentiles look correct. Either I made a mistake, or it was the T5 DNN that lowered these.